From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Tue, 15 Mar 2016 11:56:51 -0400
Subject: fix -Wformat

---
 calcApp/src/sCalcPerform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/calcApp/src/sCalcPerform.c b/calcApp/src/sCalcPerform.c
index 15610bb..66d1f8e 100644
--- a/calcApp/src/sCalcPerform.c
+++ b/calcApp/src/sCalcPerform.c
@@ -1541,7 +1541,7 @@ epicsShareFunc long
 				if (((s = strpbrk(s, "%")) == NULL) ||
 					((s = strpbrk(s+1, "*cdeEfgGiousxX")) == NULL)) {
 					/* no printf arguments needed */
-		 			sprintf(tmpstr, ps->s);
+		 			sprintf(tmpstr, "%s", ps->s);
 				} else {
 					switch (*s) {
 					default: case '*':
