From: Michael Davidsaver <mdavidsaver@gmail.com>
Date: Thu, 28 Jul 2016 13:37:13 -0400
Subject: backport: install postfix headers

---
 calcApp/src/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/calcApp/src/Makefile b/calcApp/src/Makefile
index d33c186..c3c8f6a 100644
--- a/calcApp/src/Makefile
+++ b/calcApp/src/Makefile
@@ -16,6 +16,9 @@ ifdef SSCAN
 	DBDINC += swaitRecord
 endif
 
+# publish headers for sCalc and aCalc engines
+INC += aCalcPostfix.h sCalcPostfix.h
+
 # <name>.dbd will be created from <name>Include.dbd, if it exists
 DBD_INSTALLS += calcSupport.dbd
 DBD_INSTALLS += calc.dbd
